modules/preferences/main/index.coffee

```diff
@@ -284,10 +295,10 @@ class Preferences extends hx.EventEmitter
   supportedTimezones: option 'supportedTimezones'
   timezoneOffsetLookup: option 'timezoneOffsetLookup'
 
-  applyTimezoneOffset: (date, offset) ->
```

Comment:
There was a problem with applyTimezoneOffset usage so I need to reapply utc time manually. Please check it again. If this solution is not acceptable for you I will search for another.